AEPTarget/Tests/UnitTests/TargetStateTests.swift

```diff
@@ -16,6 +16,19 @@
 import XCTest
 
 class TargetStateTests: XCTestCase {
+    override func setUpWithError() throws {
+        for _ in 0 ... 5 {
+            for key in UserDefaults.standard.dictionaryRepresentation().keys {
+                UserDefaults.standard.removeObject(forKey: key)
+            }
+        }
```

Comment:
i'm confused about the outer loop here

Reply:
Yeah, additional iterations in the outer loop ensure that the defaults are actually cleared. Also as discussed, I've updated the code to use the clear method defined in the UserDefaults extension in Target TestHelpers instead.